code/modules/antagonists/heretic/knowledge/side_flesh_void.dm

```diff
@@ -37,6 +37,23 @@
 	route = PATH_SIDE
 	depth = 8
 
+/datum/heretic_knowledge/spell/void_prison
+	name = "Void Prison"
+	desc = "Grants you Void Prison, a spell that places your victim into ball, making them unable to do anything or speak. \
+		Applies void chill afterwards"
+	gain_text = "At first, I see myself, waltzing along a snow-laden street. \
+		I try to yell, grab hold of this fool and tell them to run. \
+		But the only welts made are on my own beating fist. \
+		My smiling face turns to face me, reflecting back in glassy eyes the empty path I have been lead down."
```

Comment:
```suggestion
		My smiling face turns to regard me, reflecting back in glassy eyes the empty path I have been lead down."
```
I know I wrote this, but I think this is better than saying 'face' twice in the same line.